File: Pratica 6 - Funcoes 2/java_logic/Exercicio5_v1.java
import java.util.Scanner;

public class Exercicio5_v1 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Digite a temperatura em Celsius: ");
        double c = sc.nextDouble();
        double f = (c * 9/5) + 32;

        System.out.println("A temperatura em Fahrenheit é: " + f);

        sc.close();
    }
}

File: Pratica 6 - Funcoes 2/cpp_logic/exercicio5_v1.cpp
#include <iostream>
using namespace std;

int main() {
    float c;
    cout << "Digite a temperatura em Celsius: ";
    cin >> c;

    float f = (c * 9/5) + 32;
    cout << "A temperatura em Fahrenheit é: " << f;

    return 0;
}